apps/web/src/config/constants/endpoints.ts

```diff
@@ -45,13 +44,26 @@ export const ACCESS_RISK_API = 'https://red.alert.pancakeswap.com/red-api'
 
 export const CELER_API = 'https://api.celerscan.com/scan'
 
-export const INFO_CLIENT_WITH_CHAIN = V2_SUBGRAPHS
+export const EXPLORER_API = 'https://info-gateway.pancakeswap.com'
```

Comment:
do we need to move to the pkg as well?

Reply:
This will be only used for web. For pkg by default uses the public subgraph

Reply:
ok but public subgraph is gonna be deprecated right?

Reply:
No